scripts/date-snapshot/repos

```diff
@@ -0,0 +1,4 @@
+libtraceevent;https://git.kernel.org/pub/scm/libs/libtrace/libtraceevent.git/;libtraceevent;20220601
+libtracefs;https://git.kernel.org/pub/scm/libs/libtrace/libtracefs.git/;libtracefs;20220601
+trace-cmd;https://git.kernel.org/pub/scm/utils/trace-cmd/trace-cmd.git/;master;20220601
+kernel-shark;https://github.com/yordan-karadzhov/kernel-shark-v2.beta.git/;kernelshark;20220601
```

Comment:
This is not the correct kernelshark repo, I wonder how you even managed to find this one.

Use ths one on kernel.org:
git://git.kernel.org/pub/scm/utils/trace-cmd/kernel-shark.git

Reply:
This was the source specified in the original GitHub actions script, so I copied it over. Thanks for clarifying that we should be using the kernel.org version.